src/ol_concourse/pipelines/infrastructure/k8s_apps/docker_pulumi.py

```diff
@@ -290,11 +292,75 @@ def build_app_pipeline(app_name: str) -> Pipeline:
                 load_var="image_tag", file=f"{app_ci_image.name}/tag", reveal=True
             ),
         ],
+        additional_post_steps=[
+            TaskStep(
+                task=Identifier("purge-fastly-cache"),
+                config=TaskConfig(
+                    platform=Platform.linux,
+                    image_resource=AnonymousResource(
+                        type=REGISTRY_IMAGE,
+                        source=RegistryImage(repository="alpine/curl"),
+                    ),
+                    run=Command(
+                        path="sh",
+                        args=[
+                            "-exc",
+                            """curl -H "Fastly-Key: ((fastly.fastly_api_token))" -H "Accept: application/json" -i -X POST "https://api.fastly.com/service/((fastly.learn_service_id_ci))/purge_all" """,
```

Comment:
Should the service ID var be a dict lookup? Currently this is hardcoding to purging Learn (which is probably accurate, but brittle).